cmake/install/Findo3de.cmake.in

```diff
@@ -12,7 +12,15 @@ include(FindPackageHandleStandardArgs)
 
 # This will be called from within the installed engine's CMakeLists.txt
 macro(ly_find_o3de_packages)
-@FIND_PACKAGES_PLACEHOLDER@
+    if(LY_MONOLITHIC_GAME)
+        set(monolithic_file "${LY_ROOT_FOLDER}/cmake/Platform/${PAL_PLATFORM_NAME}/Monolithic/o3de_subdirectories_${PAL_PLATFORM_NAME_LOWERCASE}.cmake")
+        if(NOT EXISTS ${monolithic_file})
+            message(FATAL_ERROR "O3DE SDK was not generated to support monolithic builds")
+        endif()
+        include("${monolithic_file}")
+    else()
+        include("cmake/Platform/${PAL_PLATFORM_NAME}/Default/o3de_subdirectories_${PAL_PLATFORM_NAME_LOWERCASE}.cmake")
```

Comment:
We should make the path on line 22 to be consistent with line 16.

That should use LY_ROOT_FOLDER as well
```
"${LY_ROOT_FOLDER}/cmake/Platform/${PAL_PLATFORM_NAME}/Default/o3de_subdirectories_${PAL_PLATFORM_NAME_LOWERCASE}.cmake"